targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_MODULE_UBLOX_ODIN_W2/sdk/ublox-odin-w2-drivers/default_wifi_interface.cpp

```diff
@@ -15,10 +15,14 @@
  * limitations under the License.
  */
 
+#ifdef MBED_CONF_RTOS_PRESENT
```

Comment:
That should use DEVICE_WIFI

Reply:
It cannot be that because `DEVICE_WIFI` is **ALWAYS** included for that target as it is in `targets.json`.
If there is a wifi library perhaps we can add a `"present" : 1` attribute in its `mbed_lib.json`. I have not identified a `wifi` library.

Reply:
Isn't wifi dependant on lwip anyway? So would use the same as for `OdinWiFiInterface.h` file, instead of rtos, use lwip dependency.

Reply:
@0xc0170 Please review [this](https://github.com/ARMmbed/mbed-os/compare/e8e9c8fd6f430ebff9a92f8b9a5ed7abdd4e38c2..157d1267691c7f87e192e16dfc13f8204a5e4e7c).